<?xml version="1.0"?>
<launch>

    <arg name="width" default="424"/>
    <arg name="height" default="240"/>
    <arg name="fps" default="15"/>

    <node name="hmi_tracker" pkg="ur3_moveit" type="hmi_tracker.py" respawn="false" output="screen" args="demo" />
    
    <include file="$(find realsense2_camera)/launch/rs_camera.launch">
        <arg name="filters" value="pointcloud" />
        <arg name="camera" value="camera_top" />
        <arg name="depth_width" value="$(arg width)" /> 
        <arg name="depth_height" value="$(arg height)" />
        <arg name="depth_fps" value="$(arg fps)" />
        <!-- min is 256 x 144 with 90 FPS (!) but for a very small region ! -->
        <arg name="color_width" value="$(arg width)" /> 
        <arg name="color_height" value="$(arg height)" />
        <arg name="color_fps" value="$(arg fps)" />

    </include>

    <node pkg="tf2_ros" type="static_transform_publisher" name="camera_link_broadcaster" args="0 0 0.5 0.707 0 -0.707 0 world camera_top_link" />

</launch>

